pkgs/development/ocaml-modules/happy-eyeballs/default.nix

```
@@ -4,16 +4,18 @@
 
 buildDunePackage rec {
   pname = "happy-eyeballs";
-  version = "0.1.3";
+  version = "0.3.0";
 
   minimalOCamlVersion = "4.08";
 
   src = fetchurl {
     url = "https://github.com/roburio/happy-eyeballs/releases/download/v${version}/happy-eyeballs-${version}.tbz";
-    sha256 = "sha256:0ns1bxcmx0rkq4am6vl2aargdzkfhria8sfmgnh8dgzvvj93cc1c";
+    sha256 = "17mnid1gvq1ml1zmqzn0m6jmrqw4kqdrjqrdsrphl5kxxyhs03m6";
   };
 
-  propagatedBuildInputs = [
+  strictDeps = true;
+
+  buildInputs = [
```

```suggestion
  propagatedBuildInputs = [
```
"domain-name duration fmt ipaddr logs" should be propagated.